packages/extension/public/css/daily-companion-app.css

```diff
@@ -0,0 +1,8 @@
+daily-companion-app {
+  line-height: 1.5; /* 1 */
```

Comment:
Just a question, what does the number comments signify?

Reply:
To be completely honest, no idea.
It was something we had setup in the extension, and was thinking it might have been a fallback for certain old browsers?

Reply:
Oh, cool. Thought this was something new. Thank you!